server/v2/api/telemetry/config.go

```diff
@@ -1,13 +1,32 @@
 package telemetry
 
-type Config struct {
-	// Prefixed with keys to separate services
-	ServiceName string `mapstructure:"service-name" toml:"service-name" comment:"Prefixed with keys to separate services."`
+func DefaultConfig() *Config {
+	return &Config{
+		Enable:                  true,
+		Address:                 "localhost:1338",
```

Comment:
In v2, as discussed yesterday, the telemetry endpoint will be in another port than grpc gateway.